packages/nx/src/plugins/js/lock-file/yarn-parser.ts

```diff
@@ -461,10 +485,32 @@ function mapSnapshots(
         snapshotMap.get(snapshot).add(requestedKey);
       }
     }
+    const requestedResolutionsVersion = getPackageJsonVersion(
```

Comment:
Version might not match the `dependencies` block but it could match the override in `resolutions`